test/e2e/run_test.go

```diff
@@ -115,17 +115,16 @@ var _ = Describe("Podman run", func() {
 	It("podman run --signature-policy", func() {
 		session := podmanTest.Podman([]string{"run", "--pull=always", "--signature-policy", "/no/such/file", ALPINE})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
+		if IsRemote() {
+			Expect(session).To(ExitWithError(125, "unknown flag: --signature-policy"))
```

Comment:
Is there value is running this test remote if the flag doesn't exist?

Reply:
Depends: could there ever be a day in which this flag will be implemented in remote? If so, this test would fail, and indicate to the developer that the test needs tweaking.

Reply:
I think it is good to check for it, at least ensure we give out a proper error and not silently ignore it.